pallets/crowdloan-reward/Cargo.toml

```diff
@@ -13,17 +13,17 @@ targets = ["x86_64-unknown-linux-gnu"]
 
```

Comment:
According to my search, this pallet has quite a number of dependencies which don't make use of workspace inheritance.